electroncash_plugins/fusion/plugin.py

```diff
@@ -228,7 +228,7 @@ def get_target_params_1(wallet, wallet_conf, active_autofusions, eligible):
         # that are either 1) eligible or 2) being fused. (Should stay constant
         # as fusions are added/cancelled)
         n_coins = sum(len(acoins) for addr,acoins in eligible)
-        n_total = n_coins + sum(len(f.inputs) for f in active_autofusions)
+        n_total = n_coins + sum(len(getattr(f,'inputs',())) for f in active_autofusions)
```

Comment:
I started being a nazi with Jonald about python lint.  I think probably because for BCHN we lint the python and also because I installed PyCharm which yells at me when I do this. 

Would you mind terribly if I beseeched thee to add spaces after the commas?  Coming from C and C++ such things seem silly -- I remember a time when if code didn't look like line noise you weren't considered respectable.

It is what it is -- would you mine?

(And yes I know I as well haven't always done this -- but I've turned over a new Python leaf!).

Otherwise looks good.

Reply:
Hehe OK, maybe we should start to turn on some autopep8 things?

Reply:
I tried but it had a heart attack...